app/forms/feedback_form.rb

```diff
@@ -0,0 +1,67 @@
+class FeedbackForm
```

Comment:
We will need some  specs for this class

Reply:
I've added some basic tests, although this is effectively a configuration file at the moment. I'll have a look at decoupling the survey monkey template formatter from this too.